docs/how_to/render_plane_manipulator.md

```diff
@@ -19,7 +19,7 @@ the rendering plane.
      napari
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
```

Comment:
There arn't really any more details there. 
I'll put the DOI link there instead of having it be hidden. I don't think anyone would expect clicking a DOI to download anything, it's a record locator after all.